subprojects/core/src/integTest/groovy/org/gradle/api/tasks/options/TaskOptionIntegrationTest.groovy

```diff
@@ -321,4 +321,121 @@ Options
         then:
         result.assertTaskExecuted(':myTask').assertOutputContains('Serial: 4321')
     }
+
```

Comment:
And what about... (I think it should fail)
```
interface Intf1 {
    @Option(option="foobar")
    Property<String> getFoo()
}
abstract class MyTask extends DefaultTask implements Intf1 {
    @Option
    Property<String> foobar = project.objects.property(String.class);
}
```

Reply:
`option` isn't required on fields, but I think this is effectively the case you're testing in WithDuplicateOptionInAnotherInterface below, so we're good.